api/routes/api/slackImporter.js

```diff
@@ -13,11 +13,8 @@ slackRouter.get('/', (req: any, res: any) => {
   const code = req.query.code;
   const communityId = req.query.state;
   const connectedBy = req.user.id;
-  const clientUrl = req.query.clientUrl;
   const returnURI = IS_PROD
-    ? clientUrl.indexOf('alpha') >= 0
-      ? 'https://alpha.spectrum.chat/api/slack'
-      : 'https://spectrum.chat/api/slack'
+    ? 'https://spectrum.chat/api/slack'
```

Comment:
You probably just want this to be `/api/slack`? That should redirect to the URL the request orgininated from—otherwise you could also use `req.baseUrl`, which should do the same thing.

Reply:
Yeah, I reverted this since we're doing a prod cut + hot fix strategy for this anyways, but thanks for the info. Will see if I can sort this out after :)